pages/about/groups/agwg.md

```diff
@@ -33,67 +34,80 @@ Some information on this page is also shown on, and may be more current in, the
 
-These tools are used by participants of the Working Group. For ways non-participants can contribute, see [How to contribute to the Working Group and file comments](https://www.w3.org/WAI/WCAG20/comments/).
+To become a participant requires a commitment from you to support the work of the group in the following ways:
 
-### Meeting Minutes
+* Engage actively with other participants in the working group using the group’s communication channels.
+* Keep up with weekly tasks and the progress of the work &mdash; for example, via the minutes of past meetings, mailing list discussions, and GitHub issue comments.
+* Give your input on time.
 
-[Minutes from previous meetings](https://www.w3.org/WAI/GL/minutes-history) are available.
+To learn how you can become a participant in the group, see [Instructions for joining the Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/instructions/).
 
-## Administrative Information
+## Group members and task forces
 
-Work of the AG WG is in accordance with the [W3C Process](https://www.w3.org/2023/Process-20231103/). AG WG work is funded in part by the [WAI Core 2015 Project](https://www.w3.org/WAI/about/projects/wai-core-2015/). The work of this group does not necessarily reflect the views or policies of the funders.
+* [Chairs](https://www.w3.org/groups/wg/ag/participants/#chairs)
+* [Current participants](https://www.w3.org/groups/wg/ag/participants/#participants)
+* [Task forces](https://www.w3.org/groups/wg/ag/task-forces/) that support the AG Working Group
 
-The chairs of the AG WG, responsible for overall leadership and management, are Chuck Adams, Alastair Campbell, and Rachael Montgomery. The staff contact, responsible for W3C Process and general support, is [Kevin White](https://www.w3.org/People/kevin/). Administrative inquiries may be sent to [group-ag-chairs@w3.org](mailto:group-ag-chairs@w3.org).
+## Communications and decisions
 
-Work on WCAG 3 is additionally led by Shawn Lauriat and Jeanne Spellman, task force co-facilitators, and Wilco Fiers, WCAG 3 Project Manager. Inquiries about WCAG 3 should be sent to [group-ag-plan@w3.org](mailto:group-ag-plan@w3.org) to reach these people as well as the chairs.
+* [Minutes from previous meetings](https://www.w3.org/WAI/GL/minutes-history)
+* [Mailing list archive](https://lists.w3.org/Archives/Public/w3c-wai-gl/)
+* [Record of decisions](https://www.w3.org/WAI/GL/wiki/Decisions)
+* [Decision policy](/about/groups/agwg/decision-policy/)
 
-The AG WG maintains the following operational resources:
+## Publications and copyright
 
-* [Decision policy](decision-policy);
-* [Record of decisions made by the WG](https://www.w3.org/WAI/GL/wiki/Decisions);
-* [meeting minutes](https://www.w3.org/WAI/GL/minutes-history).
+* [List of publications by the AG Working Group](https://www.w3.org/groups/wg/ag/publications/)
+* [Intellectual property rights &mdash; AG Working Group](https://www.w3.org/groups/wg/ag/ipr/)
```

Comment:
This text need to remain as is or close to this:

> W3C maintains a [public list of any patent disclosures](https://www.w3.org/groups/wg/ag/ipr/) made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent.

Reply:
I've added in the text and adjusted the link to go to the relevant section on the page. 
This is so that the link text better reflects the keywords the user is expecting. 
Will this work for you?

Reply:
No, I think this needs to be previous wording or close to:

> W3C maintains a [public list of any patent disclosures](https://www.w3.org/groups/wg/ag/ipr/) made in connection with the deliverables of the group. There are also [instructions for disclosing a patent](https://www.w3.org/groups/wg/ag/ipr/#discl-howto).

Reply:
Sorry, I somehow missed that second link! That solves the problem.
I've moved the first link back to the top of the page, added the second link, and fixed small issues with the link text (placement and keywords) while keeping close to the previous wording.